web-admin/src/routes/[organization]/[project]/-/share/[token]/+page.svelte

```diff
@@ -21,8 +21,10 @@
   $: ({ data: cookieProject } = $cookieProjectQuery);
   $: if (cookieProject) {
     eventBus.emit("banner", {
+      type: "default",
       message: `Limited view. For full access and features, visit the <a href='/${organization}/${project}/${metricsView}'>original dashboard</a>.`,
       includesHtml: true,
+      iconType: "alert",
```

Comment:
This is more an informational message than a warning, so I think this should be an "info" icon. (See my other message about default icons for each `type`.)

Reply:
The file had `AlertCircleOutline`. Since icon type match the file name, alert makes sense here.